### PR TITLE
Move perl code from bin to libperl 

### PR DESCRIPTION
**Pull request by robmaz**

### CODE REVIEW

**Comment by magicDGS** [comment]
I am cleaning the branch before reviewing this change.

File: distmap [hunk 1]
```diff
@@ -6,8 +6,11 @@ use Pod::Usage;
 use File::Which;
 use File::Temp;
 
-use FindBin qw/$RealBin/;
-use lib "$RealBin/bin";
+# This is also in @INC, but leave it here so that it
+# can be easily adjusted in case the installer puts the
+# modules in some private folder
+my LibDir = "/usr/local/lib/perl5";
```
Comment (magicDGS) [inline distmap]: This means that you are installing the software in `/usr/local/`, but that's not true always. I think that using the `$RealBin` is fine too as before:

```perl
use FindBin qw/$RealBin/:
use lib "$RealBin/lib/perl5"
```

Comment (magicDGS) [reply distmap]: In addition, do you really want to store in `lib/perl5`, which is language specific? In that case, there are several scripts in the folder that are python (and even a `JDistmap.jar` that I missed in #21).

Maybe it is better to just name it `lib` as-is, and then organize by language or functionality or even differentiate modules/scripts.

Comment (robmaz) [reply distmap]: Are these non-perl things even used for anything anymore?

**Comment by magicDGS** [comment]
You push again the commits that I removed from the branch to clean it. If you want, pass by my office to show you how to update your branch and not making it dirty anymore...

**Pull request closed without merging**